scopes/component/component/component.ts

```diff
@@ -105,6 +105,10 @@ export class Component {
     }
   }
 
+  async getSnaps() {
```

Comment:
don't call this getSnaps, it's confusing, as it doesn't return snaps, but component log entries.
snaps is something else

Reply:
Makes sense, I was originally intending to call it `getLogs` but I saw the query on `ComponentHost` call it as `snaps`, so wanted to keep it consistent. 
If you are ok with it, I will rename it to `getLogs`

Reply:
getLogs is better. that's the name I thought about as well

Reply:
also, see that in the host itself (not in the gql) it is called getLogs. 
https://github.com/teambit/bit/blob/803b02d8f17b7ea66a3b90320d455e73598e52e8/scopes/component/component/component.graphql.ts#L174
I think that calling it snaps in the host gql is a mistake, and in that case, it's better to use logs in all places rather then keep consistency.
I would also add a comment in the gql schema above the host snaps saying that it returns the logs
maybe even mark the host snaps as deprecated - https://www.apollographql.com/docs/apollo-server/schema/directives/

Reply:
this is confusing. we should use one term all the way.
what are "logs" and how are they related to snaps/tags?